test/src/components/year-selector.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {render} from '@testing-library/preact';
+
+// The date needs to be set before the import
```

Comment:
Two separate tests because the date is used to set `now` upon import and I need to test different dates.